dev_docs/nav-kibana-dev.docnav.json

```diff
@@ -64,7 +64,35 @@
           "id": "kibDevAnatomyOfAPlugin"
```

Comment:
Was this file move intentional?

Reply:
Yes.  It didn't make sense to me that the `docnav` was in a different, albeit parent, folder when we tend to colocate other `docnav` definitions.

Reply:
I can revert this change, of course... but I think the relocation makes sense.

Reply:
Seems fine, wanted to double check if it needed to live in the repo root.